magicctapipe/scripts/lst1_magic/semi_automatic_scripts/config_auto_MCP.yaml

```diff
@@ -30,5 +30,5 @@ general:
     env_name: magic-lst  # name of the conda environment to be used to process data.
     cluster: "SLURM"  # cluster management system on which data are processed. At the moment we have only SLURM available, in the future maybe also condor (PIC, CNAF).
     nice:  # Set the job priority (only positive integer value). A lower nice value increases the priority, while a higher value reduces it.
-
+    allowed_M_tels: [1, 2] #MAGIC telescopes used for analysis. Default: [1,2] which coresponds to M1 and M2. Do not use the order [2,1]. The list must not be empty!
```

Comment:
you could always sort the list when you read it, then it does not matter if [1,2] or [2,1] is given